src/components/routes/AccountSettings.tsx

```diff
@@ -255,7 +256,13 @@ function AccountSettings({ me }: { me: APIUser }) {
 		}
 		<Paper elevation={2} className={classes.paper}>
 			<Typography component="h2" variant="h6" color="textPrimary" align="left" gutterBottom>Account Settings</Typography>
-			<Typography component="p" color="textSecondary" align="center" className={classes.margin}>To change any of the information here, please contact us directly.</Typography>
+			<Typography component="p" color="textSecondary" align="center" className={classes.margin}>
+				<>To change any of the information here, please </>
+				<Link component={RouterLink} to={'/contact'} variant="subtitle1" color="textSecondary">
+					contact us.
```

Comment:
```suggestion
					contact us
```

Reply:
Oops! Sorry,  I didn't notice the typo here. I'll make the change.